src/_macosx.m

```diff
@@ -1804,8 +1804,20 @@ - (void)mouseEntered:(NSEvent *)event
     NSWindow* window = [self window];
     if ([window isKeyWindow]==false) return;
 
+    int x, y;
+    NSPoint location = [event locationInWindow];
+    location = [self convertPoint: location fromView: nil];
+    x = location.x * device_scale;
+    y = location.y * device_scale;
+
     gstate = PyGILState_Ensure();
-    result = PyObject_CallMethod(canvas, "enter_notify_event", "");
+    PyObject* loc = Py_BuildValue("ii", x, y);
+    Py_INCREF(Py_None);
```

Comment:
you can just borrow the ref to None (i.e. don't bother with Py_INCREF/Py_DECREF)

Reply:
Is that because we're staying inside our function? I know we have to inc ref when returning (or use `Py_RETURN_NONE`).

Reply:
yup